.github/workflows/stale.yml

```diff
@@ -0,0 +1,34 @@
+# SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+# SPDX-License-Identifier: MIT
+name: Close stale issues
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+
+    if: ${{ github.repository_owner != 'nextcloud-gmbh' }}
+
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v9
+      with:
+        repo-token: ${{ secrets.COMMAND_BOT_PAT }}
+        stale-issue-message: >
+          Hi there! It looks like this issue hasn’t seen much activity for a while and might need a bit more information to move forward.
+          If we don’t hear back soon, we may close it to keep things organized.
+          Thanks again for your contributions – we really appreciate it!
+        stale-issue-label: 'stale'
+        only-labels: 'needs info'
+        labels-to-remove-when-unstale: 'needs info,stale'
+        exempt-issue-labels: '1. to develop,2. developing,3. to review,4. to release,discussion'
+        days-before-stale: 30
```

Comment:
Let's go higher or the action might tag too many tickets as stale in the next few days. Our issue tracker is still not fully triaged.

How about 90?

Reply:
> Let's go higher or the action might tag too many tickets as stale in the next few days. Our issue tracker is still not fully triaged.

I think that's fine because the stale action will only comment on issues with the needs info label. Everything else is ignored.